Adjust multi-cloud config to avoid extra topics (#240)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file-for-multiple-clouds.adoc

.stf-connectors.yaml
[source,yaml,options="nowrap"]
----
resource_registry:

>   OS::TripleO::Services::Collectd: /usr/share/openstack-triple-heat-templates/deployment/metrics/collectd-container-puppet.yaml    # <1>

Author: ```suggestion
  OS::TripleO::Services::Collectd: /usr/share/openstack-tripleo-heat-templates/deployment/metrics/collectd-container-puppet.yaml    # <1>
```

Author: Found this in testing, but everything else works now.

    CeilometerQdrEventsConfig:
        driver: amqp
        topic: cloud1-event   # <3>

    CeilometerQdrMetricsConfig:
        driver: amqp
        topic: cloud1-metering   # <4>

    CollectdAmqpInstances:
        cloud1-notify:        # <5>
            notify: true
            format: JSON
            presettle: false
        cloud1-telemetry:     # <6>
            format: JSON
            presettle: true

ifdef::include_when_16[]
    CollectdSensubilityResultsChannel: sensubility/cloud1-telemetry # <7>
endif::include_when_16[]
----

> <1> Directly load the collectd service due to not including collectd-write-qdr.yaml environment file for multi-cloud deployments.

Reviewer: ```suggestion
<1> Directly load the collectd service because you are not including the `collectd-write-qdr.yaml` environment file for multi-cloud deployments.
```

. Include the `stf-connectors.yaml` file and unique domain name environment file `hostnames.yaml` in the `openstack overcloud deployment` command, along with any other environment files relevant to your environment:
+

> WARNING: Use of `collectd-write-qdr.yaml` with a custom `CollectdAmqpInstances` parameter will result in both the custom and default topics being published to. In a multi-cloud environment the `stf-connectors.yaml` directly loads the collectd service via the `resource_registry` parameter.

Reviewer: ```suggestion
[WARNING]
If you use the `collectd-write-qdr.yaml` file with a custom `CollectdAmqpInstances` parameter, data publishes to the custom and default topics. In a multiple cloud environment, the configuration of the `resource_registry` parameter in the `stf-connectors.yaml` file loads the collectd service.
```
The warning was a little hard to read so slight editing. @leifmadsen will you check that I didn't change the technical correctness please?
